recipes/sirius/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "sirius" %}
+{% set version = "7.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/electronic-structure/SIRIUS/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 01bf6c9893ff471473e13351ca7fdc2ed6c1f4b1bb7afa151909ea7cd6fa0de7
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - make
+    - cmake >=3.14.0
+  host:
+    - {{ mpi }}
+    - libblas
+    - libcblas
+    - liblapack
+    - scalapack
```

Comment:
in my experience, scalapack may also need to be specified under `run` (otherwise it can be missing)

the differentiation between `host` and `run` is not entirely clear to me, even after reading the [docs](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-host-and-run) (also [here](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section))

my current understanding is this (please feel free to educate me further ;-)):
 * anything that is not a build tool should be listed under `host` (idea: you might be cross-compiling from the `build` system to the `host`)
 * shared libraries should *in principle* be listed both under `host` and under `run`.

however, at the build stage I think conda looks for linked shared libraries and adds them as dependencies, which is why one can get away with not listing those under `run` (but in the scalapack case e.g. it can fail; at least it did for me in the cp2k feedstock).

Reply:
e.g. I guess this is what gives rise to [these warnings](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=307443&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1183), and in principle the shared libraries should also be listed under `run`

Reply:
Apperently, one has to use `run_exports` to make sure a host dependency translates into a run dependency: https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#export-runtime-requirements
I tried to update spla already, and if it helps, I'll update SpFFT as well. Scalapack seems to be missing that section.

Reply:
And for whatever reason, if I add scalapack as a run dependency, cmake can no longer find python 3 at build time. This seems completely unrelated to me, so no idea why this is happening.

Reply:
Ah, now it all makes sense :-) 

Perhaps you might want to make a PR to the scalapack feedstock? ;-)